store.css

```diff
@@ -0,0 +1,109 @@
+/* main section */
+h1 {
+    color: rgb(224, 99, 58);
+    font-weight: 500;
+    margin-top: 50px;
+    text-align: left;
+}
+.logo {
+    height: 5px;
+    width: 5px;
+    float: left;
```

Comment:
Try not to use the `float` property.

I did some Googling, and it seems that it can be quite tricky to work with sometimes. (_I don't have much knowledge about it_)